### DISCUSSION TITLE
Write to chat

### DISCUSSION THREAD

**Discussion by telefon2232** [Tnze/go-mc#141] [Q&A] [answered]
Good afternoon,

I want to start new player (bot) to server. But I need to write to chat "/register password" to register. But I didn't find function for this action. Could you help me?

Thank you!

**Comment by maxsupermanhd** [comment]
You have to send a packet, see protocol docs https://wiki.vg/Protocol#Chat_Message_.28serverbound.29

**Comment by SuperiorZeik** [comment, marked as answer]
> #109
